addrmgr/addrmanager.go

```diff
@@ -1027,9 +1027,11 @@ func (a *AddrManager) HasLocalAddress(na *wire.NetAddress) bool {
 	return ok
 }
 
-// FetchLocalAddresses fetches a summary of local addresses information for
+// LocalAddresses returns a summary of local addresses information for
 // the getnetworkinfo rpc.
-func (a *AddrManager) FetchLocalAddresses() []LocalAddr {
+//
+// This function is safe for concurrent access.
+func (a *AddrManager) LocalAddresses() []LocalAddr {
```

Comment:
Unfortunately this can't be changed at the moment since it would be a major semver version break to the API and it's neither internal package nor is it using a new major that hasn't been released yet.

However, adding a function is not a major version bump, only a minor one, so what I would suggest is to keep this as the primary function, but add back `FetchLocalAddresses` defined in terms this one and then deprecate it.

```Go
// FetchLocalAddresses fetches a summary of local addresses information for
// the getnetworkinfo rpc.
//
// Deprecated: This will be removed in the next major version bump.
// Use LocalAddresses instead.
func (a *AddrManager) FetchLocalAddresses() []LocalAddr {
	return a.LocalAddresses()
}
```

Reply:
Got it, thanks! I added that in per your suggestion.